src/style.css

```diff
@@ -0,0 +1,43 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars > div {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 > div,
+.stars--1 > div:nth-child(n + 2),
+.stars--2 > div:nth-child(n + 3),
+.stars--3 > div:nth-child(n + 4),
+.stars--4 > div:nth-child(n + 5) {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```

Comment:
Set one background-image by default for general class and use nth-child only for changing it